lib/http-bridge/test/integration/Test/Integration/Scenario/Transactions.hs

```diff
@@ -95,6 +100,126 @@ spec = do
         verify ra
             [ expectEventually ctx balanceAvailable (oneMillionAda - fee - amt)
             ]
+
+    it "TRANS_CREATE_02 - Multiple Output Transaction to single wallet" $ \ctx -> do
+        widSrc <- view walletId <$> fixtureWallet ctx
+        widDest <- createWallet ctx "Test dest1" mnemonics18
+        (_, addrs1) <-
+            unsafeRequest @[ApiAddress] ctx
+                ("GET", "v2/wallets/" <> widDest <> "/addresses") Empty
+
+        let amt = 1
+        let destination1 = (addrs1 !! 1) ^. #id
+        let destination2 = (addrs1 !! 2) ^. #id
+        let payload = Json [json|{
+                "payments": [{
+                    "address": #{destination1},
+                    "amount": {
+                        "quantity": #{amt},
+                        "unit": "lovelace"
+                    }
+                },
+                {
+                    "address": #{destination2},
+                    "amount": {
+                        "quantity": #{amt},
+                        "unit": "lovelace"
+                    }
+                }],
+                "passphrase": "cardano-wallet"
+            }|]
+        let fee = 168653
+
+        --post transaction
+        r <- request @ApiTransaction ctx
+            ("POST", "v2/wallets/" <> widSrc <> "/transactions")
+            Default payload
+        verify r
+            [ expectResponseCode HTTP.status202
```

Comment:
I don't know why this request is giving me `403`. Everything works fine with `curl`....

Reply:
you can't do multi-output transactions at the moment with those faucet wallets. That's something I wanted to enable yesterday but time flies. Since there's only a single UTxO on the faucet wallet at the moment, you can only have a single output recipient :) 
We need to send more than one output on the faucet wallet and have let's say, one million ADA, but not as 1x1M, but 10x100_000 !